eo-phi-normalizer/app/Main.hs

```diff
@@ -227,7 +230,7 @@ cliOpts =
 
 data StructuredJSON = StructuredJSON
   { input :: String
-  , output :: [[String]]
+  , output :: [[(String, String)]]
```

Comment:
Stringly-typed programming :) Need to clean this up (we can do it in a separate PR).